specification/appendix-d-observability.md

```diff
@@ -19,49 +19,55 @@ This is particularly relevant to telemetry-related [hooks](./sections/04-hooks.m
+| `feature_flag.version`    | `version`         | `Recommended`     | `string` | A version string (format unspecified) for the flag or [flag set](./glossary.md#flag-set).                                                                                                       |
 
 > [!NOTE]
 > Keys in flag metadata use the "camelCase" casing convention, while the OpenTelemetry standard uses a namespaced "snake_case" convention.
 
 ### Provider Metadata
 
-| Log Record Attribute         | Provider Metadata Field | Requirement level | Notes                                                                                            |
-| ---------------------------- | ----------------------- | ----------------- | ------------------------------------------------------------------------------------------------ |
-| `feature_flag.provider_name` | `name`                  | `Recommended`     | The name of the provider as defined in the `provider metadata`, available in the `hook context`. |
+| Log Record Attribute         | Provider Metadata Field | Requirement level | Type     | Notes                                                                                            |
+| ---------------------------- | ----------------------- | ----------------- | -------- | ------------------------------------------------------------------------------------------------ |
+| `feature_flag.provider.name` | `name`                  | `Recommended`     | `string` | The name of the provider as defined in the `provider metadata`, available in the `hook context`. |
+
+## History
+
+Feature flags in the OpenTelemetry semantic conventions are currently in development and are marked as experimental.
+The following table describes the history of changes to the OpenTelemetry feature flag log records as it progresses towards a stable release.
+
+| Original Field Name                     | New Field Name                | Semantic Convention Release                                                            |
+| --------------------------------------- | ----------------------------- | -------------------------------------------------------------------------------------- |
+| `feature_flag.variant`                  | `feature_flag.result.variant` | [v1.32.0](https://github.com/open-telemetry/semantic-conventions/releases/tag/v1.32.0) |
+| `feature_flag.evaluation.reason`        | `feature_flag.result.reason`  | [v1.32.0](https://github.com/open-telemetry/semantic-conventions/releases/tag/v1.32.0) |
+| `feature_flag.evaluation.error.message` | `error.message`               | [v1.33.0](https://github.com/open-telemetry/semantic-conventions/releases/tag/v1.33.0) |
+| `feature_flag.provider_name`            | `feature_flag.provider.name`  | [v1.33.0](https://github.com/open-telemetry/semantic-conventions/releases/tag/v1.33.0) |
+| `value`                                 | `feature_flag.result.value`   | Unreleased |
```

Comment:
The value change is still pending but should be approved soon. I'm comfortable releasing this spec change prior to it being officially in the OTel semcon because it's only a matter of time before it's added. I'll update the table once it has been included in a release version.